src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -341,6 +341,13 @@ unsigned BackdropNodeGadget::layerMask() const
 	return (unsigned)GraphLayer::Backdrops;
 }
 
+Imath::Box3f BackdropNodeGadget::renderBound() const
+{
+	// If it's important to render text that extends outside the backdrop, we should include the header bound
+	// here - but we don't have access to style here?
```

Comment:
We can access that with `this->style()`.

Reply:
Getting this right ended up being pretty annoying, so for the moment I've just made the comment more accurate.